src/Microsoft.OData.Edm/Validation/ValidationRules.cs

```diff
@@ -2485,18 +2485,37 @@ public static class ValidationRules
                 (context, annotation) =>
                 {
                     IEnumerable<EdmError> errors;
-                    if (!annotation.Value.TryCast(annotation.Term.Type, out errors))
+                    if (!annotation.UsesDefault)
                     {

```

Comment:
should not need this once we fix it so that annotation.Value always returns a value (either explicit or default)

Reply:
Done ✅